### PR TITLE
Feedback Feature

### PR DESCRIPTION
**Pull request by AlmightyMikkel**

### CODE REVIEW

File: Runtime/Game/LootLockerSDKManager.cs [hunk 1]
```diff
@@ -5350,6 +5350,139 @@ public static void GetRemovedUGCForPlayer(GetRemovedUGCForPlayerInput input, Act
...
+            SendFeedback(LootLockerFeedbackTypes.player, ulid, description, category_id, onComplete);
+        }
+
+        /// <summary>
+        /// Sends a feedback with the given data, will return 204 upon successful request.
+        /// </summary>
+        /// <param name="description">Reason behind the report</param>
+        /// <param name="category_id">A unique identifier of what catagory the report should belong under</param>
+        /// <param name="onComplete">onComplete Action for handling the response of type LootLockerResponse</param>
+        public static void SendGameFeedback(string description, string category_id, Action<LootLockerResponse> onComplete)
```
[the remark this answers is not in the document]
Comment (AlmightyMikkel) [reply Runtime/Game/LootLockerSDKManager.cs:5451]: I'm not sure I follow :/ Isn't that what I'm doing

[the remark this answers is not in the document]
Comment (AlmightyMikkel) [reply Runtime/Game/LootLockerSDKManager.cs:5451]: Ah, sorry I forget that not all the relevant info will be in the "commented" on, There is no reason other than I missed it!

[the remark this answers is not in the document]
Comment (kirre-bylund) [reply Runtime/Game/LootLockerSDKManager.cs:5451]: I'm just confused as to why you resolved the comment without answering :P Did you disagree?

[the remark this answers is not in the document]
Comment (kirre-bylund) [reply Runtime/Game/LootLockerSDKManager.cs:5415]: Fixed now

**Comment by kirre-bylund** [comment]
Needs to be rebased before merging

**Review by kirre-bylund — APPROVED** [review]

**Pull request merged**